LICENSE-binary

```diff
@@ -237,7 +237,6 @@ org.apache.spark:spark-network-shuffle_2.12
 org.apache.spark:spark-sketch_2.12
 org.apache.spark:spark-tags_2.12
 org.apache.spark:spark-unsafe_2.12
-commons-httpclient:commons-httpclient
```

Comment:
Sorry. Hadoop-2.7 need this:
```
[INFO] +- org.apache.hadoop:hadoop-client:jar:2.7.4:compile
[INFO] |  +- org.apache.hadoop:hadoop-common:jar:2.7.4:compile
[INFO] |  |  +- commons-cli:commons-cli:jar:1.2:compile
[INFO] |  |  +- xmlenc:xmlenc:jar:0.52:compile
[INFO] |  |  +- commons-httpclient:commons-httpclient:jar:3.1:compile
```

Reply:
oh well ... seems still can't get rid of this then. :(

Reply:
We can also move the dependency under `hadoop-3.2` profile so Spark won't package it by default, although I'm not sure whether that helps much with the issue the PR is trying to address.

Reply:
Sorry, I don't understand the above comment, can you please elaborate on this?

Reply:
Spark currently support two Hadoop profiles: `hadoop-2.7` and `hadoop-3.2`. Since this library is only used in the former, we could remove it from the latter so it at least won't be used and distributed. You can check `hadoop-3.2` profile in `pom.xml` to see how it handles Hadoop 3.x specific dependencies.